src/cli/gentest/templates/blockchain_test/transaction.py.j2

```diff
@@ -8,37 +8,36 @@ from typing import Dict
 
 import pytest
 
-from ethereum_test_tools import Account, Block, BlockchainTestFiller, Environment, Transaction
+from ethereum_test_tools import (
+    Account,
+    Block,
+    BlockchainTestFiller,
+    Environment,
+    Storage,
+    Transaction,
+)
 
 REFERENCE_SPEC_GIT_PATH = "N/A"
 REFERENCE_SPEC_VERSION = "N/A"
 
 
 @pytest.fixture
 def env():  # noqa: D103
-    return Environment(
-{{ environment_kwargs }}
-    )
-
+    return {{ environment  | stringify }}
 
 @pytest.mark.valid_from("Paris")
```

Comment:
Not a requirement right now but it would be interesting that, if we have the fork schedule for the network from which we are obtaining the transaction, we can put the correct fork in here.

It would not work for devnets for example, but at least for mainnet and public testnets it's doable.

Reply:
Hey Mario - I don't quite follow this. Could you elaborate on a new issue? I will take it up separately. This PR has become quite big.

Reply:
Basically, right now every test starts from the `Paris` fork, but it could be that the fork that was active when the transaction was executed was a newer fork, and therefore the transaction could execute differently or not even work in `Paris` because, e.g., some opcode that it uses was not enabled until after `Paris`.